adoc/chapters/programming_interface.adoc

```diff
@@ -12349,6 +12349,17 @@ implementation-defined object of unspecified type, which is interpreted by
 [code]#parallel_for# to construct an appropriate [code]#reducer#
 type as detailed in <<reducer-class>>.
 
+If an identity value is supplied to a reduction, an implementation may use that
```

Comment:
```suggestion
If an identity value is supplied to a reduction, an implementation will use that
```

Think we need to say the identity you give won’t just be ignored.

Reply:
I've had a go at addressing this in b7fb2d6f3e59940078f0e5b8a7c94c08facee546.  I don't think it's as simple as replacing "may" with "will", because an implementation isn't required to use temporary accumulator variables at all -- but if an implementation does use temporary variables, they should be initialized to the identity.

Reply:
That change LGTM.